templates/dbconfig.h2.xml.erb

```diff
@@ -0,0 +1,23 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
could you convert this to an epp() template please?

Reply:
I figured consistency is better, otherwise we have to convert all the templates to EPP, and if we bother doing that, we should just make it one template and conditionally include the lines as needed instead of having several files that are almost exactly the same.

Reply:
@bastelfreak What do you think?

Reply:
ah good point, let's keep it as is.